vars/main.yml

```diff
@@ -29,10 +30,17 @@ _bootstrap_packages:
   CentOS:
     - python2
     - sudo
+  Fedora-30:
```

Comment:
It looks like you have been working on this for quite a while, this was implemented a while back.